From cd2a61e208894d17bc9d02713b1d2b29e7e13c06 Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:36:02 +0200
Subject: Tests: ignore the failure to write uncompressed data

Ignore the 'Unable to write uncompressed print data: Broken pipe'  error that
sometimes breaks the errorlines counting on kfreebsd for some reason.
Origin: vendor
Last-Update: 2016-08-09

Patch-Name: tests-ignore-kfreebsd-unable-to-write-uncompressed-print-data.pdf
---
 test/run-stp-tests.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index 9c970b27b..b081eb58e 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -989,6 +989,7 @@ count=`$GREP '^E ' $BASE/log/error_log | \
        $GREP -v 'loadFile failed: temp file: not a PDF file' | \
        $GREP -v 'Failed to connect to system bus' | \
        $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | \
+       $GREP -v 'Unable to write uncompressed print data: Broken pipe' | \
        wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
